.gitlab-ci.yml

```diff
@@ -0,0 +1,39 @@
+image: ubuntu:20.04
+
+stages:
+  - build
+  - release
+
+build:
+  stage: build
+  script:  
+    - apt update
+    - DEBIAN_FRONTEND="noninteractive" apt install -y git autoconf automake autotools-dev cmake curl libmpc-dev libmpfr-dev libgmp-dev gawk build-essential bison flex texinfo gperf libtool patchutils bc zlib1g-dev libexpat-dev wget byacc device-tree-compiler python gtkwave vim-common virtualenv python-yaml
+    - mkdir dist
+    - make build_sdk_lite -j$(nproc)
+    - tar -czvf dist/sdk_lite.tgz install
+    - make build_sdk -j$(nproc)
+    - tar -czvf dist/sdk.tgz install prog lib ucode include bedrock perch bootrom bp-demos bp-tests
+    - make build_prog -j$(nproc)
+    - tar -czvf dist/prog.tgz install prog lib ucode include bedrock perch bootrom bp-demos bp-tests riscv-tests coremark beebs
+  only:
+    - tags
+  timeout: 3 hours
+  artifacts:
+    paths:
+      - dist
+
+release:
+  stage: release
+  script:
+    - apt update
+    - apt install -y software-properties-common
+    - apt-key adv --keyserver keyserver.ubuntu.com --recv-key C99B11DEB97541F0
```

Comment:
What's this key? Is this supposed to be secret?

Reply:
No it's just the recommended way of installing github cli on ubuntu.
https://github.com/cli/cli/blob/trunk/docs/install_linux.md#debian-ubuntu-linux-apt

That being said if we were to use github actions instead of gitlab-ci then we could skip that installation.
I just saw that you were using gitlab-ci for the main black-parrot repo so I also used it here.
Not sure if you do have a preference for one tool or the other...